test/case/convention/openshift/golang-osd-operator/01_generated_files_checker

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -x
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+source $REPO_ROOT/test/lib.sh
+
+repo=$(empty_repo)
+add_cleanup $repo
+test_project="file-generate"
+
+# TODO: test REPO_NAME detection by setting an appropriately-formatted
+# git remote
+export REPO_NAME=${0##*/}
+
+bootstrap_project $repo ${test_project} openshift/golang-osd-operator
+cd $repo
+export GOPATH=$GOPATH:`pwd`
+
+make update_boilerplate
+make check-generate
+if [ $? = 0 ] ; then 
+    echo "No error detected while checking on uncommitted project"
+    exit 1
+else
+    echo "Project is not committed. Checks failed as expected"
+fi
+
+git add -A
+git commit -m "initial commit"
+
+echo $GOPATH
+
+make check-generate
+if [ $? = 0 ] ; then 
+    echo "Error not detected while the generated files have not beent added/committed"
+    exit 1
+else
+    echo "Generated files are not committed. Checks failed as expected"
```

Comment:
```suggestion
    echo "Generated files are committed. Checks succeeded as expected"
```
?

Reply:
No. Initial commit does not contain the files to be generated (as the is_clean check is failling, file generation is not triggered by the first `check-generate`). 
As a consequence, this step is generating new files which are not committed in git, and so, the check is expected to fail due to the existing diff.
The 3rd check (after the commit including the generated files, l.47 ) is testing the check is passing when everything is generated/committed.

Reply:
Right, got it, thanks.